src/mscorlib/src/System/IO/TextWriter.cs

```diff
@@ -606,30 +606,69 @@ public virtual Task FlushAsync()
         [Serializable]
         private sealed class NullTextWriter : TextWriter
         {
-            internal NullTextWriter(): base(CultureInfo.InvariantCulture) {
-            }
-        
-            public override Encoding Encoding {
-                get { return Encoding.Default;  }
+            internal NullTextWriter() : base(CultureInfo.InvariantCulture)
+            {
             }
 
-            [SuppressMessage("Microsoft.Contracts", "CC1055")]  // Skip extra error checking to avoid *potential* AppCompat problems.
-            public override void Write(char[] buffer, int index, int count) {
-            }
-            
-            public override void Write(String value) {
-            }
+            public override Encoding Encoding => Encoding.Unicode;
```

Comment:
Slipped past me that the original implementation was returning `Encoding.Default`; my bad.